xc7/make/arch_define.cmake

```diff
@@ -21,7 +21,7 @@ function(ADD_XC7_ARCH_DEFINE)
     DEVICE_FULL_TEMPLATE \${DEVICE}-\${PACKAGE}
     CELLS_SIM ${YOSYS_DATADIR}/xilinx/cells_sim.v ${symbiflow-arch-defs_SOURCE_DIR}/xc7/techmap/cells_sim.v
     VPR_ARCH_ARGS "\
-      --clock_modeling route \
+      --clock_modeling dedicated_network \
```

Comment:
I don't believe this change is suitable at this time?

Reply:
I don't think this will work without a clock network model in arch.xml.

Reply:
Restored the original version.